### PR TITLE
create genre screen

### PR DESCRIPTION
**Pull request by trantan97**
## [Pull request title]
Create genre screen
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7959
#### Screenshots (nếu có)
image

### CODE REVIEW

**Comment by trantan97** [comment]
ready

File: app/src/main/java/com/ptit/filmdictionary/ui/category/CategoryActivity.java [hunk 1]
```diff
@@ -2,139 +2,55 @@
...
-        mBinding.setViewModel(mViewModel);
-        mViewModel.loadMoviesByCategory(mViewModel.getPage());
+    protected void loadMoreMovies() {
+        hideLoadMore(false);
+        int nextPage = mViewModel.getPage();
+        ++nextPage;
+        mViewModel.loadMoviesByCategory(nextPage);
     }
 
     public static Intent getInstance(Context context, String categoryKey, String categoryTitle) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/ptit/filmdictionary/ui/category/CategoryActivity.java]:
```suggestion
    public static Intent getIntent(Context context, String categoryKey, String categoryTitle) {
```

File: app/src/main/java/com/ptit/filmdictionary/ui/genre/GenreActivity.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    @Override
+    protected void loadMoreMovies() {
+        hideLoadMore(false);
+        int nextPage = mViewModel.getPage();
+        ++nextPage;
+        mViewModel.loadMoviesByGenre(nextPage);
+    }
+
+    public static Intent getInstance(Context context, String genreKey, String categoryTitle) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/ptit/filmdictionary/ui/genre/GenreActivity.java]:
```suggestion
    public static Intent getIntent(Context context, String genreKey, String categoryTitle) {
```

**Comment by trantan97** [comment]
ready

**Pull request merged**